dataload/import_to_elasticsearch.py

```diff
@@ -483,6 +520,106 @@ def to_band(value, bins, labels):
         previous_value = b
 
 
+def update_doc_with_other_locations(grant):
+    """ This flattens out some embedded data for easier indexing """
+
+    # Prior versions of additional_data may not have this field
+    # or if locationLookup failed entirely for this grant
+    if not grant["additional_data"].get("locationLookup"):
+        return
+
+    for location in grant["additional_data"]["locationLookup"]:
+        # beneficiaryLocation
+        if location["source"] == "beneficiaryLocation":
+
+            if not grant["additional_data"].get("GNBeneficiaryDistrictName"):
+                try:
+                    grant["additional_data"]["GNBeneficiaryDistrictName"] = location["ladnm"]
+                    grant["additional_data"]["GNBeneficiaryDistrictGeoCode"] = location["ladcd"]
+                except KeyError:
+                    pass
+
+            if not grant["additional_data"].get("GNBeneficiaryCountyName"):
+                try:
+                    grant["additional_data"]["GNBeneficiaryCountyName"] = location["utlanm"]
+                except KeyError:
+                    pass
+
+            if not grant["additional_data"].get("GNBeneficiaryRegionName"):
+                try:
+                    grant["additional_data"]["GNBeneficiaryRegionName"] = location["rgnnm"]
+                    grant["additional_data"]["GNBeneficiaryRegionGeoCode"] = location["rgncd"]
+                except KeyError:
+                    try:
+                        # Wales, Scotland, Northern Ireland don't have rgnm so we set them as country
+                        grant["additional_data"]["GNBeneficiaryRegionName"] = location["ctrynm"]
+                        grant["additional_data"]["GNBeneficiaryRegionGeoCode"] = location["ctrycd"]
+                    except KeyError:
+                        pass
+
+        # recipientOrganisationLocation
```

Comment:
```suggestion
        # recipientOrganizationLocation
```

just for consistency..